md/migrations/0003_agency_slug.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django_extensions.db.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('md', '0002_auto_20160623_1209'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='agency',
+            name='slug',
+            field=django_extensions.db.fields.AutoSlugField(editable=False, populate_from='name', null=True, blank=True),
```

Comment:
needs to include max_length=255 to correspond to nc, otherwise defaults to 50, which is too short.